zppy/templates/cryosphere.cfg

```diff
@@ -0,0 +1,8 @@
+[e3sm_diags]
+sets = "lat_lon","zonal_mean_xy","zonal_mean_2d","polar","cosp_histogram","meridional_mean_2d","enso_diags","qbo","diurnal_cycle","annual_cycle_zonal_mean","streamflow"
+
+[e3sm_diags_vs_model]
+sets = "lat_lon","zonal_mean_xy","zonal_mean_2d","polar","cosp_histogram","meridional_mean_2d"
+
```

Comment:
@wlin7, could you take a look at this list or flag someone to take a look?  I would like to make sure we're running the analysis that you would want for cryosphere runs.

If there are other zppy config options you would like to change from the defaults for cryosphere runs, here is the place to add them.

If you would like to change other things that zppy doesn't yet support, this PR would be a good place to make those changes, too.  (As long as they're not too complicated.)

Reply:
Thanks for creating this template, @xylar. The sets for e3sm_iags vs observations are the typical onesI haven't seen what are included in 'streamflow'.

For e3sm_diags_vs_model, I would like to see 'enso_diags', and 'annual_cycle_zonal_mean' also included, for easy comparison of model differences. These could be more useful when comparing cryo simulations against WC simulations.

Reply:
Thanks @wlin7.  I'll make these changes.